### PR TITLE
Add summarization

### PR DESCRIPTION
**Pull request by edouard-sn**
#### Description

Adds summarization route, request and request in protorepo.

#### Changelog

- [ENHANCEMENT] Summarize route in `recommendations.proto`

### CODE REVIEW

File: noted/recommendations/v1/recommendations.proto [hunk 1]
```diff
@@ -24,3 +25,11 @@ message ExtractKeywordsBatchRequest {
 message ExtractKeywordsBatchResponse {
     repeated ExtractKeywordsResponse keywords_array = 1;
 }
+
+message SummarizeRequest {
+    string content = 1;
+}
+
+message SummarizeResponse {
+    string result = 1;
```
Comment (rojas-diego) [inline noted/recommendations/v1/recommendations.proto:34]: I'm having trouble understanding what will be stored within `result`. I feel like a more precise variable name would help here. Is `result` a bunch of keywords? a structured string? something else?

**Review by rojas-diego — CHANGES_REQUESTED** [review]
Nice! Also small note, each and every commit should build correctly so I think in the future you should add the endpoint and the response/request pair within a single commit.

File: noted/recommendations/v1/recommendations.proto (hunk 1 above)
Comment (edouard-sn) [reply noted/recommendations/v1/recommendations.proto:34]: We can call it "summary" if that's better

Comment (rojas-diego) [reply noted/recommendations/v1/recommendations.proto:34]:
```suggestion
    // Small text output which summarizes the original entry.
    string summary = 1;
```
WDYT?

**Pull request merged**